File: atcoder/abc192/abc192_a.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main{
    
    public static void main(String[] args) {
        BufferedReader bf = new BufferedReader(new InputStreamReader(System.in));
        try {
            int coins = Integer.parseInt(bf.readLine());
            System.out.println(100-(coins % 100));
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: atcoder/abc192/abc192_a.kt
import java.io.BufferedReader
import java.io.InputStreamReader

fun main()
{
    val In = BufferedReader(InputStreamReader(System.`in`))
    val numero = In.readLine()
    var decs = "0"
    decs = if(numero.toInt() >= 10)
    {
        numero[numero.length-2] + numero[numero.length-1].toString()
    }
    else {
        numero
    }
    println(100-decs.toInt())
}